### PR TITLE
Feature/bound 11 create marshall first turn

### PR DESCRIPTION
**Pull request by SCustodero**
Won't pull this one because the cargames file shouldn't be included, but we'll re-pull it.

### CODE REVIEW

**Review by georgiok741 — CHANGES_REQUESTED** [review]
Looks great Sean! You have a lot of great stuff here. For now, lets get rid of line 9-12, since we will need to find a way to integrate it with the main file and pull from specific split decks. Otherwise, this looks great!

**Comment by SCustodero** [comment]
> Looks great Sean! You have a lot of great stuff here. For now, lets get rid of line 9-12, since we will need to find a way to integrate it with the main file and pull from specific split decks. Otherwise, this looks great!

Thanks bro. I'll get rid of those lines so we can use your functions.

File: source/marshall_setup.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        draw_card = deck.getCard()
+        marshall.addCard(draw_card, isKnown=True)
+
+    marshall.showHand()
+
+    #Display board
+    
+    while(True):
+        guess_all = input("Would you like to guess all fugitive locations? (y/n)")
+        if guess_all.lower() == 'y' or guess_all.lower() == 'yes':
```
Comment (youraveragedeveloper7) [inline source/marshall_setup.py:20]: Great work Sean! The only stylistic change I would put is to use the lower() method on the input itself; then you won't have to use the lower method on the guess_all variable multiple times throughout the loop.

**Review by youraveragedeveloper7 — CHANGES_REQUESTED** [review]

File: source/marshall_setup.py (hunk 1 above)
Comment (SCustodero) [reply source/marshall_setup.py:20]: That's a great point, thanks big dawg.

**Review by youraveragedeveloper7 — APPROVED** [review]
Looks great! Thanks for consolidating your feature under the main CardGames.py file.

**Review by georgiok741 — APPROVED** [review]

**Pull request merged**